src/main/kotlin/org/eclipse/uprotocol/transport/validator/UAttributesValidator.kt

```diff
@@ -1,52 +1,45 @@
-/*
- * Copyright (c) 2024 General Motors GTO LLC
+/**
+ * SPDX-FileCopyrightText: 2024 Contributors to the Eclipse Foundation
  *
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
  *
- *   http://www.apache.org/licenses/LICENSE-2.0
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
  *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- * SPDX-FileType: SOURCE
- * SPDX-FileCopyrightText: 2023 General Motors GTO LLC
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package org.eclipse.uprotocol.transport.validate
+package org.eclipse.uprotocol.transport.validator
 
-import org.eclipse.uprotocol.uri.validator.validate
-import org.eclipse.uprotocol.uri.validator.validateRpcMethod
-import org.eclipse.uprotocol.uri.validator.validateRpcResponse
+import org.eclipse.uprotocol.uri.validator.isDefaultResourceId
+import org.eclipse.uprotocol.uri.validator.isRpcMethod
+import org.eclipse.uprotocol.uri.validator.isRpcResponse
 import org.eclipse.uprotocol.uuid.factory.getTime
 import org.eclipse.uprotocol.uuid.factory.isUuid
 import org.eclipse.uprotocol.v1.*
 import org.eclipse.uprotocol.validation.ValidationResult
 
 
 /**
- * [UAttributes] is the class that defines the Payload. It is the place for configuring time to live, priority,
+ * [UAttributes] is the class that defines the Payload. It is the place
+ * for configuring time to live, priority,
  * security tokens and more.
- * Each UAttributes class defines a different type of message payload. The payload can represent a simple published
+ * Each UAttributes class defines a different type of message payload. The
+ * payload can represent a simple published
```

Comment:
@dddj698 can you fix the formatting of the header (the line breaks) in the various methods in this class?

Reply:
will do in next PR